src/buttons/EditButton/EditButton.tsx

```diff
@@ -1,41 +1,59 @@
 import EditIcon from '@mui/icons-material/Edit';
+import { ListItemIcon, MenuItem } from '@mui/material';
 import IconButton, { IconButtonProps } from '@mui/material/IconButton';
 import Tooltip from '@mui/material/Tooltip';
 
 import React, { FC, MouseEventHandler } from 'react';
 
+import { ActionButton, ActionButtonVariant } from '../../types';
+
 export type Props = {
   id?: string;
   className?: string;
   ariaLabel?: string;
   onClick?: MouseEventHandler;
   tooltip?: string;
   size?: IconButtonProps['size'];
+  type?: ActionButtonVariant;
 };
 
 const EditButton: FC<Props> = ({
   id,
   className,
   ariaLabel,
   onClick,
-  tooltip = 'edit',
+  tooltip = 'Edit',
```

Comment:
Nitpick: Now that we use the `tooltip` prop as the text prop I think it could be renamed to something more meaningful ? `label` maybe ? I know this will be a breaking change but since you will have to refactor anyway, may be it would be worth it to improve the component's API